### PR TITLE
Prepare protractor helper to work with async await

### PR DESCRIPTION
**Pull request by wlsf82**
## Description

This PR updates the lib to be prepared to work with async/await, and it is related to issue #21.

## Pull Request Checklist

- [x] Have you followed the steps in our [Contributing](../../blob/master/docs/CONTRIBUTING.md#steps-to-contribute) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
- [x] Have you successfully run tests with your changes locally?
- [x] Have you written new tests for your changes, if necessary? _(No new tests were needed.)_
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](../../blob/master/docs/) folder and [README.md](../../blob/master/README.md))
- [x] Have you linked the correct [GitHub issue](../../issues), if there is any?

## Other comments

- I suggest reviewing commit by commit.
- This other issue (#22) may need to be implemented together in this PR, to fix the Better Code Hub analysis, or we could simply do this in another PR, which I'm in favor of.

### CODE REVIEW

**Comment by wlsf82** [comment]
@PauloGoncalvesBH, o BCH falhou, mas revise mesmo assim, please.

File: README.md [hunk 1]
```diff
@@ -93,32 +94,30 @@ The same test could be written as below, using the protractor-helper library.
...
   });
 });
 ```
 
-As you can see, by using the protractor-helper library the code is easier to read. Also, there is no need of unnecessary complexity.
+As you can see, by using the protractor-helper library the code is shorter and easier to read. Also, there is no need of unnecessary complexity.
+
+## Pre-requirements
+
+- Node.js v8+ (due to the use of async/await)
```
Comment (PauloGoncalvesBH) [inline README.md]: Protractor version 6.0.0 isn't also a pre-requirement?

Comment (wlsf82) [reply README.md]: I've executed the existing tests after the changes with version 5.4.2 of Protractor, and all tests have still passed, so I don't think we need that.

Comment (wlsf82) [reply README.md]: Please resolve this conversation if you are ok with that.

**Comment by PauloGoncalvesBH** [comment]
Suggestion: Add the keyword **testing** on _package.json_ file.
This is necessary to be one of the projects suggested in the topic 'testing' of [npmjs.com](npmjs.com).

**Comment by wlsf82** [comment]
@PauloGoncalvesBH, I've just pushed the code with your suggestion. Back to you.

**Comment by PauloGoncalvesBH** [comment]
> @PauloGoncalvesBH, I've just pushed the code with your suggestion. Back to you.

The commit daa8e68 is ok also.
 We need analyze the BCH result.

**Comment by PauloGoncalvesBH** [comment]
> * This other issue (#22) may need to be implemented together in this PR, to fix the Better Code Hub analysis, or we could simply do this in another PR, which I'm in favor of.

 Let's make this adjustment in another PR.

**Comment by wlsf82** [comment]
I'm closing this PR without merging since it's super outdated and it has too many conflicts. I may use some of its implementations in the new one when addressing issue #21 again.

**Pull request closed without merging**